documentation/firmware-management/index.rst

```diff
@@ -87,32 +86,29 @@ Upon discovery the Firmware Version property becomes read-only to prevent human
 ============================================
 Who can access Firmware Management features?
 ============================================
-As of Hyperview 3.4, any user can access Firmware Management features as long as:
-
-* You are using a Firmware Management-licensed instance (see :ref:`View license<View-license-doc>`.
-* You have access to at least one asset with managed firmware.
+As of Hyperview 3.4, any user can access Firmware Management features as long as you are using a Firmware Management-licensed instance (see :ref:`View license<View-license-doc>`.
```

Comment:
Missing the closing parenthesis. Should be: (see :ref:`View license<View-license-doc>`).

Reply:
Thanks! Fixed ..